docs/pbs/index.md

```diff
@@ -313,21 +313,50 @@ so if you will want to submit the same script at times from Casper,
 *always append the server name to your queue*.
 
 ### Interactive Jobs
+Interactive jobs provide an interactive session on a compute node, useful for debugging, testing code, and running short tasks that require user interaction.
 
-Users can start an interactive job on `casper` or `Derecho` using the `qsub -I` command. The `-I` flag is used to request an interactive job. The following example shows how to start an interactive job on `casper`:
+Users can start an interactive job on `casper` or `Derecho` using the `qsub -I` command. The `-I` flag is used to request an interactive session. The following example shows how to start an interactive job with specified resources on `casper`:
 
 ```bash
 qsub -I -l select=1:ncpus=1:mem=20GB -q casper@casper-pbs -l walltime=06:00:00 -A <project_code>
 ```
 
-Additionally, the `qinteractive` command can be used to start an interactive job on `derecho` or `casper`. For `derecho` this will start an interactive job on the `develop` queue, while for `casper` it will submit the job to the `main` queue. The following example shows how to start an interactive job on `derecho` or `casper`:
+
+#### `qinteractive`
+
+The `qinteractive` command provides a convenient way to start an interactive job on either `derecho` or `casper`. By default, when you run `qinteractive` on casper, it starts an interactive job with 1 CPU and 10GB of memory. On Derecho, `qinteractive` starts an interactive job with 32 CPUs and 55GB of memory on the `develop` queue.
+
+The following example shows how to start an interactive job on either `derecho` or `casper` :
 
 ```bash
-qinteractive -A <project_code>
+qinteractive
```

Comment:
Is there a reason you dropped the `-A <project_code` part? Users will need to provide that unless they set the `PBS_ACCOUNT` environment variable, which I believe was not documented yet on this page.

Reply:
I have added `-A <project_code>` part and a tip box including information about setting project codes via `PBS_ACCOUNT` and `DAV_PROJECT`.